test/HybridCustody_tests.cdc

```diff
@@ -617,27 +616,38 @@ pub fun testGetNFTsAccessibleFromProxyAccount(){
 
     let expectedChildIDs = (scriptExecutor("example-nft/get_ids.cdc", [child.address]) as! [UInt64]?)!
     let expectedParentIDs: [UInt64] = []
-    let expectedAddressToIDs: {Address: [UInt64]} = {parent.address: expectedParentIDs, child.address: expectedChildIDs}
+    let expectedAddressToIDs: {Address: [UInt64]} = {child.address: expectedChildIDs, parent.address: expectedParentIDs}
```

Comment:
This line is fine, but commenting about the test name. There are a few others I found with "proxy" in the name:

- `testGetProxyAccountFTCapabilities`
- `testGetPublicCapabilityFromProxy`

Reply:
Ah good catch, thought I found these